### DISCUSSION TITLE
Ability to change the default End of Line Sequence

### DISCUSSION THREAD

**Discussion by MikeBrasser** [nuxtrdev/nuxtr-vscode#87] [Questions]
If I make a new vue component/page or whatever it uses LF as default End of Line sequence. I would like to change the default value to CRLF. Currently I don't see an option for that. I've already configured my project correctly so when I manually create .vue file it will use the End of Line sequence CRLF.

**Comment by adhamfarrag** [comment]
Hi @MikeBrasser, What issues are you facing with using `\n` or LF ? Help me see the value of this feature request.

**Reply by MikeBrasser** [reply]
@adhamfarrag Some people, like myself, use certain prettier/eslint configurations that work best with CRLF. With LF it breaks compatibilty. I reckon this is the case for more people so I think this would be a great addition.

**Reply by adhamfarrag** [reply]
@MikeBrasser Thanks for the explanation. 

I can detect the end of line configuration and automatically change the style. Will that be sufficient for you? Trying to avoid introducing a new option.

**Reply by MikeBrasser** [reply]
@adhamfarrag I think that would be sufficient, thanks! For most people this would be an excellent fix.

**Reply by adhamfarrag** [reply]
@MikeBrasser You will have this coming with next release. Thanks for bringing it up. 

Anything else you'd like to have implemented or improved?

**Reply by MikeBrasser** [reply]
@adhamfarrag I have nothing on my mind right now, but I'm really looking forward to the next release :).

Just started my own project so the upcoming weeks so I'll fully experience your extension.

Thanks for this excellent extension & when I encounter something that could be improved l'll notify you ;)! 👍

**Comment by adhamfarrag** [comment]
Hi @MikeBrasser, Version 0.2.10 should include dynamic detection for eof according to `files.eof` vscode configuration. Please, update.
